package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "jQuery-MD5",
+  "original-version": "0.0.1",
```

Comment:
Although this is the first package.json file, the version of the tool should be 1.2.1